toolkit/tools/imagecustomizerlib/imagecustomizer.go

```diff
@@ -0,0 +1,231 @@
+// Copyright (c) Microsoft Corporation.
```

Comment:
Having a clean separation between the CLI and the actual business logic is goodness.